app/services/get_data.rb

```diff
@@ -0,0 +1,96 @@
+class GetData
+  HTTP_STATUS_CODES = {
+    100 => 'Continue',
+    101 => 'Switching Protocols',
+    102 => 'Processing',
+    103 => 'Early Hints',
+    200 => 'OK',
+    201 => 'Created',
+    202 => 'Accepted',
+    203 => 'Non-Authoritative Information',
+    204 => 'No Content',
+    205 => 'Reset Content',
+    206 => 'Partial Content',
+    207 => 'Multi-Status',
+    208 => 'Already Reported',
+    226 => 'IM Used',
+    300 => 'Multiple Choices',
+    301 => 'Moved Permanently',
+    302 => 'Found',
+    303 => 'See Other',
+    304 => 'Not Modified',
+    305 => 'Use Proxy',
+    306 => '(Unused)',
+    307 => 'Temporary Redirect',
+    308 => 'Permanent Redirect',
+    400 => 'Bad Request',
+    401 => 'Unauthorized',
+    402 => 'Payment Required',
+    403 => 'Forbidden',
+    404 => 'Not Found',
+    405 => 'Method Not Allowed',
+    406 => 'Not Acceptable',
+    407 => 'Proxy Authentication Required',
+    408 => 'Request Timeout',
+    409 => 'Conflict',
+    410 => 'Gone',
+    411 => 'Length Required',
+    412 => 'Precondition Failed',
+    413 => 'Payload Too Large',
+    414 => 'URI Too Long',
+    415 => 'Unsupported Media Type',
+    416 => 'Range Not Satisfiable',
+    417 => 'Expectation Failed',
+    421 => 'Misdirected Request',
+    422 => 'Unprocessable Entity',
+    423 => 'Locked',
+    424 => 'Failed Dependency',
+    425 => 'Too Early',
+    426 => 'Upgrade Required',
+    428 => 'Precondition Required',
+    429 => 'Too Many Requests',
+    431 => 'Request Header Fields Too Large',
+    451 => 'Unavailable for Legal Reasons',
+    500 => 'Internal Server Error',
+    501 => 'Not Implemented',
+    502 => 'Bad Gateway',
+    503 => 'Service Unavailable',
+    504 => 'Gateway Timeout',
+    505 => 'HTTP Version Not Supported',
+    506 => 'Variant Also Negotiates',
+    507 => 'Insufficient Storage',
+    508 => 'Loop Detected',
+    509 => 'Bandwidth Limit Exceeded',
+    510 => 'Not Extended',
+    511 => 'Network Authentication Required'
+  }
+
+  CODE_OK = HTTP_STATUS_CODES.select { |key, value| value == 'OK' }.keys
+  def info_genres
+    responsegenre = Faraday.get "https://api.themoviedb.org/3/genre/movie/list?api_key=#{ENV["API_KEY"]}&language=en-US"
+    self.handle_response(responsegenre.status)
+    return responsegenre
+  end
+
+  def info_films
+    responsefilms = Faraday.get "https://api.themoviedb.org/3/discover/movie?api_key=#{ENV["API_KEY"]}&language=en-US&sort_by=popularity.desc&include_adult=false&include_video=false&page=1&with_watch_monetization_types=flatrate"
+    self.handle_response(responsefilms.status)
+    return responsefilms
+  end
+
+  def info_film_by_id(id)
+    responsefilm = Faraday.get "https://api.themoviedb.org/3/movie/#{id}?api_key=#{ENV["API_KEY"]}&language=en-US"
+    self.handle_response(responsefilm.status)
+    return responsefilm
+  end
+
+  private
+
+  def handle_response(responsecode)
+    if responsecode == CODE_OK
+      puts HTTP_STATUS_CODES[200]
```

Comment:
Use `Rails.logger` for purposes like these, standard output usually hard to get and nobody reads